content/common/makefile/linux.mk

```
@@ -0,0 +1,16 @@
+# Get the relative path to the directory of the current makefile.
+MAKEFILE_DIR = $(dir $(lastword $(MAKEFILE_LIST)))
+include $(MAKEFILE_DIR)/defs.mk
+
+SRCS = $(wildcard *.c)
+OBJS = $(SRCS:.c=.o)
+
+$(LOGGER_OBJ): $(LOGGER_DIR)/log.c
+	@make -C $(LOGGER_DIR) $(LOGGER_OBJ) 
```

```suggestion
	@make -C $(LOGGER_DIR) $(LOGGER_OBJ)
```
Remove trailing whitespace.